osquery/numeric_monitoring/plugins/tests/filesystem_tests.cpp

```diff
@@ -0,0 +1,92 @@
+/**
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under both the Apache 2.0 license (found in the
+ *  LICENSE file in the root directory of this source tree) and the GPLv2 (found
+ *  in the COPYING file in the root directory of this source tree).
+ *  You may select, at your option, one of the above-listed licenses.
+ */
+
+#include <osquery/numeric_monitoring/plugins/filesystem.h>
```

Comment:
nit: we typically use `"` for relative paths, and order includes from broadest scope to narrowest, so push this down below, and bring the boost/gtest includes up above the SDKs.